corehq/apps/app_manager/xform.py

```diff
@@ -946,7 +946,8 @@ def get_languages(self):
         return list(self.translations().keys())
 
     def get_questions(self, langs, include_triggers=False,
-                      include_groups=False, include_translations=False, form=None):
+                      include_groups=False, include_translations=False,
+                      for_report=False):
```

Comment:
`for_report` is incongruous with the other options here, how about `include_lookups` or `include_itemsets`?

Reply:
@orangejenny `include_select_itemsets`?

Reply:
updated, used. `exclude_select_itemsets`

Reply:
finally, decided for `exclude_select_with_itemsets`.

Only if naming was easy....

Reply:
ha, agreed naming is hard. I like `include_select_itemsets` since the others are all `include`, though the default is different for this one so I don't feel that strongly.

Reply:
found `exclude` better to keep logic simple later.